tests/test_functional.py

```diff
@@ -31,7 +30,7 @@ def __exit__(self, exc_type, exc_value, traceback):
         return exc_type is RuntimeError and str(exc_value) == "App Exited"
 
 
-class FunctionalParserTests(unittest.TestCase):
+class FunctionalParserTests(unittest.TestCase):    
```

Comment:
Nitpick: can you please remove the trailing whitespace?

Reply:
No problem. I ran black on the test files to make sure they conformed to the standards. 
Would you like me to add that the to the tox configuration too?

Reply:
Thanks! If you don't mind adding a `black --check` call to the tox configuration that would be appreciated.

Reply:
odd it is failing on GitHub, but it passed locally.

Reply:
Got it to work..
Odd issue it must be that that wildcards work on Windows but not Linux. 
Checking the directory is a cleaner approach anyways.